shelley/chain-and-ledger/executable-spec/src/BlockChain.hs

```diff
@@ -143,7 +180,12 @@ slotsPrior = 33 -- one third of slots per epoch
 startRewards :: Slot.Duration
 startRewards = 33 -- see above
 
-verifyVrf :: VrfProof a => Keys.VKey -> Seed -> Proof a -> Bool
+verifyVrf
+  :: (DSIGNAlgorithm dsignAlgo, VrfProof a)
+  => VKey dsignAlgo
+  -> Seed
+  -> Proof dsignAlgo a
+  -> Bool
```

Comment:
doesn't `cardano-crypto-class` also provide a VRF (mock) implementation?

Reply:
Yeah, I'll put in another PR after this one for VRF :)